### ISSUE TITLE
stutter while scrolling albums & songs

### ISSUE THREAD

**Issue by dtirer** [BLeeEZ/amperfy#310]
version: 1.2.1 (3)
phone: iPhone 12 mini
os: 17.5.1

- All artwork is cached (I had tapped "Download all artworks in library")
- Artwork Download settings are set to "Download only once"

Anytime I scroll up and down my "Albums" or "Songs" list, the view stutters a lot.  It does not happen on the "Artists" view.

The attached video is of me just "throwing" the view up & down.  Any pauses are the "stutter" i'm describing

https://github.com/user-attachments/assets/753b0075-4e8d-4f37-82e6-0c80cbd30d53

**Comment by Schlaubischlump** [comment]
Hi @dtirer,

Does this error still occur in the latest version?

**Comment by dtirer** [comment]
> Hi @dtirer,
> 
> Does this error still occur in the latest version?

Yes this does seem to still be happening on version 1.2.2, build 3
